test/cli/test/mixed/kyverno-test.yaml

```diff
@@ -0,0 +1,19 @@
+name: ondemand
+policies:
+  - policy.yaml
+resources:
+  - resource.yaml
+results:
+  - policy: ondemand
+    rule: ondemand-nodeselector
+    resource: nodeselector
+    patchedResource: patched-resource.yaml
+    namespace: user-space
+    kind: Pod
+    result: pass
+  - policy: ondemand
+    rule: ondemand-managed_by
+    resource: nodeselector
+    namespace: user-space
+    kind: Pod
+    result: pass
```

Comment:
Thanks for adding the tests @viveksahu26 :) Could we also add some resource examples here that have a result of `fail`?

Reply:
Ok, I will add.